liota/dccs/iotcc.py

```diff
@@ -335,10 +376,26 @@ def set_properties(self, reg_entity_obj, properties):
         else:
             entity = reg_entity_obj.ref_entity
 
-        log.info("Properties defined for resource {0}".format(entity.name))
+        def on_response(msg):
```

Comment:
A lot of code seems to be common in on_response. Can't we extract it out as a function, instead of repeating code everywhere? If we have transaction id validation, and "result"=succeeded check, it should be enough.